src/main/kotlin/com/cognifide/gradle/aem/common/instance/InstanceUrl.kt

```diff
@@ -67,6 +68,7 @@ class InstanceUrl(raw: String) {
         return config.userInfo?.split(":")
                 ?.takeIf { it.size == 2 }
                 ?.get(index)
+                ?.let { decode(it) }
```

Comment:
clue of change

Reply:
Happy you fixed that, I can remember at least 3 situations when this was pain.